app/controllers/register-service.controller.js

```diff
@@ -175,35 +210,34 @@ module.exports = {
    * @param req
    * @param res
    */
-  submitOtpResend: function submitOtpResend (req, res) {
+  submitOtpResend: async function submitOtpResend (req, res) {
+    const sessionData = req.register_invite
     const correlationId = req.correlationId
-    const code = req.register_invite.code
+    const code = sessionData.code
```

Comment:
Similar to the above on trusting the session is well formed for this to not NPE https://github.com/alphagov/pay-selfservice/pull/2308#discussion_r497490462